lib/plugins/aws/deploy/compile/events/apiGateway/lib/methods.js

```diff
@@ -72,6 +121,11 @@ module.exports = {
             }
           `;
 
+          const allowOrigin = '"method.response.header.Access-Control-Allow-Origin"';
+          const corsMethodResponseTemplate = `${allowOrigin} : ${allowOrigin}`;
+          const corsMethodIntegrationTemplate =
+          corsEnabled ? `${allowOrigin}: "'${cors.origins.join(',')}'"` : '';
+
```

Comment:
Seems like this code snippet runs whether there's cors configured or not. So even for endpoints that doesn't have cors enabled they'd have this `ResponseParameters`:
```
"method.response.header.Access-Control-Allow-Origin" : "method.response.header.Access-Control-Allow-Origin"
```

Maybe I'm missing something, but this looks a bit hacky. If no cors are configured, this header shouldn't exist in the response right?